cloud/pkg/devicecontroller/types/device.go

```diff
@@ -106,3 +106,14 @@ type DeviceTwinUpdate struct {
 	BaseMessage
 	Twin map[string]*MsgTwin `json:"twin"`
 }
+
+type DeviceStateUpdate struct {
+	BaseMessage
+	DDevice `json:"device"`
+}
+
+type DDevice struct {
```

Comment:
DDevice  -> Device ?

Reply:
It will cause a name conflict with Device structure in this go file, so I named DDevice temporary.